### ISSUE TITLE
ChatGPT Functionality

### ISSUE THREAD

**Issue by wrussell1999** [MLH-Fellowship/orientation-project-js-23.FAL.A#6]
https://github.com/MLH-Fellowship/orientation-project-python/issues/13

**Comment by Zxun2** [comment]
I'll like to work on this!

**Comment by am831** [comment]
Hi @Zxun2, I'm working on the backend for this issue. Would you like to set up a time to collaborate on this? Also the link here is broken, it should take you here: https://github.com/MLH-Fellowship/orientation-project-python-23.FAL/issues/10

**Comment by Zxun2** [comment]
@am831 Yup, sure! How does 10am EST (GMT-4) tomorrow sounds? If not, I'm available after 6pm EST (GMT-4)! On discord 🫡?

**Comment by am831** [comment]
@Zxun2 my discord name is alisha.maddy, let's chat there! 6pm-8:30pm EST tomorrow works for me

**Comment by Zxun2** [comment]
@am831 Great! My discord name is zongxun. See you!
